integration_tests/integration/sex.spec.ts

```diff
@@ -0,0 +1,49 @@
+import Page from '../pages/page'
+import ImprisonmentStatusPage from '../pages/imprisonmentStatus'
+import Role from '../../server/authentication/role'
+import expectedArrivals from '../mockApis/responses/expectedArrivals'
+import SexPage from '../pages/sexPage'
+
+const expectedArrival = expectedArrivals.court.notCurrent
+const expectedArrivalWithMaleGender = expectedArrivals.withFemaleGender
+
+context('Sex', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn', Role.PRISON_RECEPTION)
+    cy.task('stubPrison', 'MDI')
+    cy.task('stubAuthUser')
+    cy.task('stubUserCaseLoads')
+    cy.task('stubExpectedArrival', expectedArrival)
+    cy.task('stubImprisonmentStatus')
+  })
+
+  it("Should display prisoner's name", () => {
+    cy.signIn()
+    const sexPage = SexPage.goTo(expectedArrival.id)
+    sexPage.prisonerName().should('contain.text', 'Sam Smith')
+  })
+
+  it('Selecting Male or Female takes user to /imprisonment-status page', () => {
+    cy.signIn()
+    const sexPage = SexPage.goTo(expectedArrival.id)
+    sexPage.sexRadioButtons('M')
```

Comment:
We've already got thorough test coverage for the top two tests here so think they can be deleted - we should be proactively looking to reduce the amount of tests in integration tests wherever possibly as they're very slow.

Specs really need to carry their own weight